app/src/main/java/com/fivegmag/a5gmsmediasessionhandler/MediaSessionHandlerMessengerService.kt

```diff
@@ -376,15 +376,55 @@ class MediaSessionHandlerMessengerService() : Service() {
         sendingUid: Int,
         provisioningSessionId: String
     ) {
-        val cacheControlHeader = headers.get("cache-control") ?: return
-        val cacheControlHeaderItems = cacheControlHeader.split(',')
-        val maxAgeHeader = cacheControlHeaderItems.filter { it.trim().startsWith("max-age=") }
+        var periodByMaxAgeHeader: Long = -1
+        var periodByExpiresHeader: Long = -1
 
-        if (maxAgeHeader.isEmpty()) {
-            return
+        val cacheControlHeader = headers.get("cache-control")
+        if(null != cacheControlHeader)
+        {
```

Comment:
Reformat the code and keep the existing rules: The `{` should be in the same line:

` if (null != cacheControlHeader) {` 

Please fix this for all of your code

Reply:
done